packages/fungible-token/bridge-fungible-token/tests/utils/setup.rs

```diff
@@ -484,15 +488,15 @@ pub(crate) fn parse_output_message_data(
 pub(crate) async fn contract_balance(
     provider: Provider,
     contract_id: &Bech32ContractId,
-    asset: AssetId,
+    asset: FuelsAssetId,
```

Comment:
Why we had to rename this one? `FuelsAssetId`?

Reply:
Because the `abigen!(...)` is putting another `AssetId` in the scope - thus a collision happens.